### ISSUE TITLE
Layers present in .gpkg, but not appearing in QRAVE

### ISSUE THREAD

**Issue by dominique-shore** [Riverscapes/QRAVEPlugin#95]
When opening an RCAT project with QRAVE, you cannot display the vwIgos and vwReaches output layers at the same time. They will both load with symbology, but only if there isn't the other output layer on the map.

**Comment by KellyMWhitehead** [comment]
I was able to open a project and the layers all seemed to load without issue. 

Were the layer names (in the layer tree) previously the same? If so, this is probably related to #126. I should have a fix for this out soon.

**Comment by dominique-shore** [comment]
Yes - we changed the layer names and are able to see them at the same time now.

**Comment by philipbaileynar** [comment]
This is related/duplicate of #126. Closing.
